x/fundraising/spec/02_state.md

```diff
@@ -97,7 +97,7 @@ type BaseAuction struct {
 
 // AllowedBidder defines a bidder who is allowed to bid with max number of bids.
 type AllowedBidder struct {
-    Bidder  string  // a bidder who is allowed to bid
+	Bidder  string  // a bidder who is allowed to bid
 	MaxBidNumber uint64  // a maximum number of bids per bidder
```

Comment:
I think `MaxBidNumber` must be `MaxBidAmount`

Reply:
I changed based on the comment.